crates/sui-indexer/src/models/transactions.rs

```diff
@@ -29,7 +29,10 @@ use crate::types::IndexerResult;
 pub struct StoredTransaction {
     pub tx_sequence_number: i64,
     pub transaction_digest: Vec<u8>,
+    /// BCS serialized TransactionData
```

Comment:
You cannot change the meaning of this column in place -- existing readers (e.g. the GraphQL service) will start to fail because they expect it to be a `SenderSignedData` and the Indexer has started to write out serialized `TransactionData` instead.

I think you have two options:

- Add a new column for the `raw_transaction_data`, backfill it, switch over to it, and then remove the old column.
- Create a whole new table to act as the key-value table for transactions.

I was looking at the columns we have in `transactions` so far and it is not very far off being its own key-value store, which means I would probably favour adding a new column rather than creating an entirely new table, but curious to get @lxfind's thoughts on that.